install.sh

```diff
@@ -1,9 +1,55 @@
 #! /bin/sh
 
+sudo dnf upgrade --refresh
+
+sudo dnf install \
+  # Alacritty on Fedora - https://github.com/alacritty/alacritty/blob/master/INSTALL.md#fedora
+  cmake freetype-devel fontconfig-devel libxcb-devel libxkbcommon-devel g++ \
+  # Better ls
+  exa \
+  # Better grep
+  ripgrep \
+  # Neovim
+  neovim python3-neovim \
+  # Better bash
+  fish \
+  # Basic i3
+  i3 i3status dmenu i3lock \
+  # Enable configuring keyboard repeat in X
+  xset \
+  # We gotta sleep sometime
+  redshift \
+  # Bluetooth tray in i3
+  blueman \
+  # Gotta give this a try but I don't have it working yet
+  sway
+
+
+# Setup Neovim
+mkdir ~/.config/nvim
+cp ./init.vim ~/.config/nvim/
 cp ./.vimrc ~/.vimrc
+
+# Setup tmux
 cp ./.tmux.conf ~/.tmux.conf
-cp ./alacritty.yml ~/.config/alacritty/alacritty.yml
+
+# Install Rust
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+cargo install alacritty starship
```

Comment:
Actually, while I love Rust and everything ... do I _really_ need these? Do these _really_ do anything? I should investigate:
1. using a default terminal (really the only thing here I need is font size and saving to clipboard by highlighting?)
2. using `fish_prompt` to set up the directory, git status, and command duration